frontend/src/metabase/palette/components/PaletteResultItem.unit.spec.tsx

```diff
@@ -1,9 +1,17 @@
-import { render, screen } from "__support__/ui";
-import { color } from "metabase/lib/colors";
+import { Route } from "react-router";
+
+import {
+  fireEvent,
```

Comment:
Are we able to use UserEvent to do our clicking instead? I think that generally the preferred way to click things in tests